uint8 leg
float64 hipAngle
float64 femurAngle
float64 tibiaAngle